diagnostics.yml

```diff
@@ -33,459 +31,338 @@ pr:
-                continueOnError: true
-                condition: always()
-
-              - task: PublishBuildArtifacts@1
-                displayName: Publish Bundled Tools
-                inputs:
-                  publishLocation: Container
-                  pathtoPublish: '$(Build.SourcesDirectory)/artifacts/bundledtools'
-                  artifactName: BundledTools
-                continueOnError: true
-                condition: always()
-
-          - template: /eng/common/templates/job/publish-build-assets.yml
-            parameters:
-              configuration: Release
-              dependsOn: Sign_Package_Publish
-              publishUsingPipelines: true
-              pool:
-                name: NetCore1ESPool-Internal
-                demands: ImageOverride -equals windows.vs2022.amd64
-
-      - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-        - template: /eng/common/templates/post-build/post-build.yml
+        - template: /eng/common/templates-official/job/publish-build-assets.yml
           parameters:
-            # This is to enable SDL runs part of Post-Build Validation Stage.
-            # as well as NuGet, SourceLink, and signing validation.
-            # The variables get imported from group dotnet-diagnostics-sdl-params
-            publishingInfraVersion: 3
-            enableSourceLinkValidation: true
-            enableSigningValidation: false
-            enableSymbolValidation: false
-            enableNugetValidation: true
-            symbolPublishingAdditionalParameters: '/p:PublishSpecialClrFiles=false'
-            publishInstallersAndChecksums: true
-            SDLValidationParameters:
-              enable: true
-              continueOnError: true
-              params: ' -SourceToolsList @("policheck","credscan")
-              -ArtifactToolsList @("binskim")
-              -BinskimAdditionalRunConfigParams @("IgnorePdbLoadError < True","Recurse < True")
-              -TsaInstanceURL $(_TsaInstanceURL)
-              -TsaProjectName $(_TsaProjectName)
-              -TsaNotificationEmail $(_TsaNotificationEmail)
-              -TsaCodebaseAdmin $(_TsaCodebaseAdmin)
-              -TsaBugAreaPath $(_TsaBugAreaPath)
-              -TsaIterationPath $(_TsaIterationPath)
-              -TsaRepositoryName "diagnostics"
-              -TsaCodebaseName "diagnostics"
-              -TsaPublish $True'
-              artifactNames:
-              - 'Packages'
+            configuration: Release
+            dependsOn: Sign_Package_Publish
+            publishUsingPipelines: true
+
+      - template: /eng/common/templates-official/post-build/post-build.yml@self
+        parameters:
+          # This is to enable SDL runs part of Post-Build Validation Stage.
+          # as well as NuGet, SourceLink, and signing validation.
+          # The variables get imported from group dotnet-diagnostics-sdl-params
```

Comment:
Juan thought the those parameters are not needed anymore by the new official build yml.  This step is still needed.

Reply:
Right, I think if we no longer need `eng/common/templates/post-build/postbuild.yml` now that we are using the official template, we can probably do some cleanup (it can be in a follow-up pr) by removing that template and removing all of the `SDLValidationParameters` references in `eng/common/templates-official/post-build/post-build.yml`